demo/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+</adaptive-icon>
```

Comment:
Are we concerned about missing the new line at end of file?

Reply:
Yep 4992e4e2c42fe2cda29159a3581834774ea3cfbb